in_toto/verifylib.go

```diff
@@ -770,9 +770,10 @@ func SubstituteParameters(layout Layout,
 
 	parameters := make([]string, 0)
 
+	re := regexp.MustCompile("^[a-zA-Z0-9_-]+$")
```

Comment:
I'm generally not a fan of `init` functions we could take this a step further and compile this once in an `init` versus each time this function is run

Reply:
We could do similar here: https://github.com/bdun1013/in-toto-golang/blob/ite7-cleanup/in_toto/model.go#L105

Reply:
Eh... moving it to a global takes it further away from the usage.  Since I'm sure this regexp compile isn't impacting performance in a meaningful its not worth changing on second thought.